::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 29 Oct 2018 16:10:48 +0100
Subject: [PATCH] IA: Rename power_password to power_pass

Align with MaaS expected variable naming after upstream change [1].

[1] https://github.com/salt-formulas/salt-formula-maas/commit/6096333

Change-Id: I0d4af1af1c54b40bf9a84ccf5f16847b9d39d456
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 config/installers/fuel/pod_config.yml.j2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/installers/fuel/pod_config.yml.j2 b/config/installers/fuel/pod_config.yml.j2
index ad40689..edb63ee 100644
--- a/config/installers/fuel/pod_config.yml.j2
+++ b/config/installers/fuel/pod_config.yml.j2
@@ -27,7 +27,7 @@ parameters:
             mac: {{ node.interfaces[nm.idx_admin].mac_address }}
           power_parameters:
             power_address: {{ node.remote_management.address.rsplit('/')[0] }}
-            power_password: {{ node.remote_management.pass }}
+            power_pass: {{ node.remote_management.pass }}
             power_type: {{ node.remote_management.type }}
             power_user: {{ node.remote_management.user }}
           architecture: {{ node.node.arch | dpkg_arch }}/generic
